src/index.css

```diff
@@ -1,8 +1,25 @@
-/* ./src/index.css */
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+
+.home-btn {
+  @apply bg-white  text-gray-900 w-40 py-2 font-light shadow-hover hover:text-gray-100 rounded-lg hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-600 focus:ring-opacity-50;
+}
+
+.nav-btn {
+  @apply bg-blue-600 text-gray-100 w-40 py-3 flex items-center rounded-lg justify-center shadow-bottom font-light hover:bg-blue-500 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:ring-opacity-50;
+}
+
+.add-btn {
+  @apply bg-green-600 text-gray-100 w-40 py-3 mt-8 hover:shadow-hover text-center rounded-lg shadow-bottom font-light hover:bg-green-500 focus:outline-none focus:ring-2 focus:ring-green-700 focus:ring-opacity-50;
+}
```

Comment:
I like this new design of add item button!

Reply:
Great way of cleaning up the html and abstracting away the CSS details!